hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -13,63 +13,69 @@ <h1 class="formH1">Dashboard</h1>
                     <p>Unfortunately, the deadline to apply for {{ event_name }} was {{ localtime(registration_close_date).strftime("%B %-d, %Y") }}.</p>
                     <br />
                     <p>If you have any questions or concerns, please <a href="mailto:{{ contact_email }}" class="hoverLink primaryText">contact us</a>.</p>
                 {% else %}
-                    <h4 class="formH2">Complete your application</h4>
+                    <h2 class="formH2">Complete your application</h2>
                     <p><strong>Status</strong>: Incomplete</p>
+                    <br />
                     <a
                         href="{{ url("registration:application") }}"
-                        class="btn-small waves-effect waves-light secondaryColor colorBtn center"
+                        class="btn-small waves-effect waves-light colorBtn"
+                        style="margin-right: auto;"
                     >
                         Get started
                     </a>
                 {% endif %}
             {% else %}
-                <h4 class="formH2">Your application has been submitted!</h4>
+                <h2 class="formH2">Your application has been submitted!</h2>
                 <p><strong>Status</strong>: Submitted</p>
+                <br />
                 <p>We're in the process of reviewing your application. Watch for an email from {{ from_email }}, and check back here for updates.</p>
             {% endif %}
-
         </div>
 
         {% if registration_open or application is not none %}
         <div class="borderTopDiv z-depth-3">
-            <h4 class="formH2">Apply as team</h4>
+            <h2 class="formH2">Apply as team</h2>
             <p>Have friends you want to work with? Create a team with up to 4 people and we’ll review your applications together.</p>
+            <br />
             {% if application is none %}
                   <p><strong>You must complete your application before you can form a team. Return here once you've submitted your application.</strong></p>
             {% else %}
                 <p>Your team code is: <strong>{{ application.team.team_code }}</strong>.</p>
                 {% if registration_open %}
                     {% set num_members = application.team.applications.count() %}
                     <p>
-                        Spots remaining on your team: <strong>{{ application.team.MAX_MEMBERS - num_members }}</strong>.
                         {% if join_team_form and num_members < application.team.MAX_MEMBERS%}
                             Share your team code with your teammates, or join their team instead.
                         {% endif %}
                     </p>
                 {% endif %}
-                <h5>Team Members</h5>
-                <ul>
+
+                <br />
+                <h3 class="formH3">Team members ({{ num_members }}/{{ application.team.MAX_MEMBERS }})</h3>
```

Comment:
Do we want to title case headers? "Team Members (3/4)" vs "Team members (3/4)". Lower case looks a little strange to me